### PR TITLE
add RISC-V updates

### PR DESCRIPTION
**Pull request by bukinr**

### CODE REVIEW

File: 2018q4/riscv.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## FreeBSD/RISC-V update
+
+Contact: Ruslan Bukin <br@freebsd.org>
+
+Contact: Mark Johnston <markj@freebsd.org>
+
+FreeBSD/RISC-V is getting mature during last quarter.
```
Comment (debdrup) [inline 2018q4/riscv.md]: "getting more mature" is more correct, I believe.

File: 2018q4/riscv.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+We have optimizied RISC-V copyin(9)/copyout(9) routines. They now supports word-sized copies where possible to dramatically increase speed of copying data between kernel and userspace.
```
Comment (debdrup) [inline 2018q4/riscv.md]: s/supports/support/

File: 2018q4/riscv.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+We made a series of improvements and bug fixes to pmap support (machine-dependent portion of virtual memory subsystem). This part was almost not touched during last years and now gets attention.
```
Comment (debdrup) [inline 2018q4/riscv.md]: I think this should be "not touched during the last years, and is now getting attention".

File: 2018q4/riscv.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+RISC-V GENERIC kernel gets support for witness(4) (The FreeBSD lock validation facility).
+
+The british company [Embecosm](https://www.embecosm.com/) has reported that they were able to boot FreeBSD on a real hardware -- SiFive Unleashed board. The support is limited to a single core only. We are expecting patches from them.
```
Comment (debdrup) [inline 2018q4/riscv.md]: s/british/British/ and "on a real hardware" needs to be "on real hardware -- a SiFive unleashed board".

**Comment by bukinr** [comment]
Thanks. Fixed :)

**Comment by debdrup** [comment]
Merged, closing #45

**Pull request merged**